packages/react/.storybook/preview.js

```diff
@@ -190,14 +191,28 @@ const decorators = [
       setRandomKey(Math.floor(Math.random() * 10));
     }, [locale, dir]);
 
+    const markdownComponentsStyles = {
+      background: 'var(--cds-background)',
+      padding: '1rem',
+    };
```

Comment:
@IgnacioBecerra I'll open a PR against this branch as an alternative way to do this.